src/core/build_target.go

```diff
@@ -701,6 +704,52 @@ func (target *BuildTarget) ExportedDependencies() []BuildLabel {
 	return ret
 }
 
+// RuntimeDependencies returns any run-time dependencies of this target.
+//
+// Although run-time dependencies are transitive, RuntimeDependencies only returns this target's direct run-time
+// dependencies. Use IterAllRuntimeDependencies to iterate over the target's run-time dependencies transitively.
+func (target *BuildTarget) RuntimeDependencies() []BuildLabel {
+	target.mutex.RLock()
+	defer target.mutex.RUnlock()
+	ret := make(BuildLabels, 0, len(target.dependencies))
+	for _, info := range target.dependencies {
```

Comment:
why `info` and not `dep`?

Reply:
It's the name of the loop variable in `ExportedDependencies` immediately above it, which is essentially identical to this one - although the function above _that_, `BuildDependencies`, is also essentially identical but uses `deps` as the loop variable name. The reason the former uses `info` is because the various `dependencies` fields are of type `[]*depInfo`, but I think `deps` is clearer so I'll rename the variable.